.changeset/good-toes-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react": patch
```

Comment:
Would this be a breaking change for customers since the breakpoint unit got removed?

Reply:
I was thinking about this too. It felt wrong to leave the breakpointUnit because it doesn't work correctly with anything except for `px`. I don't think it's a breaking change because it's not documented and unlikely to have been used. At the worst the breakpointUnit design token value would be ignored. Finally, the actual breakpoint values also haven't changed, just converted to their px values, so app behavior shouldn't change. Does that makes sense?

Reply:
That's right. We have no reason to leave it there for it is misleading.